### ISSUE TITLE
rsi calculation

### ISSUE THREAD

**Issue by jimgtchou** [Lokraan/Hasami#10]
I really appreciate you sharing your code on calculating RSI, I could be wrong but I think there may be something weird going on with the RSI formula since the constant 'rsi_time_frame' was not used anywhere in the code. It looks like the code is calculating RSI for the entire history of the crypto-pairs.

I think the first iteration should be 
https://github.com/Lokraan/Hasami/blob/d2a423c1d729b7a8ee9b7cbf00f9198df25e8cbd/hasami.py#L262
`n = self._rsi_time_frame` 

Also should become
https://github.com/Lokraan/Hasami/blob/d2a423c1d729b7a8ee9b7cbf00f9198df25e8cbd/hasami.py#L266-L268
```python
average_gain = sum(gain[0:n]) / n

average_loss = sum(loss[0:n]) / n
```

But still need to do the weighting for after 2nd+ iteration `average_gain = (old_gain*(n-1)+new_gain)/n` 

I'm new to open source projects and not sure about contribution etiquettes, would you mind for me to contribute and help to improve this project?

**Comment by Lokraan** [comment]
Sorry I was out of town but great catch I forgot to add that for some reason.

If you would like to contribute and fix it, I would greatly appreciate that.

**Comment by Lokraan** [comment]
Closed by commit f62aaac
